### ISSUE TITLE
Does CrossEntropyWithSoftMax alternative exist?

### ISSUE THREAD

**Issue by robinhad** [Microsoft/CNTK#2310]
CrossEntropy With Softmax documentation tells that target vector should sum to 1. I want to have output after activation function like [1, 1]. How can I achieve that?

**Comment by cha-zhang** [comment]
You can write your own loss functions in Python with built-in ops. For instance: 

train_loss = -C.reduce_sum(C.element_times(target, C.log(prediction)))

**Comment by n17s** [comment]
You can just give the output to sigmoid and then you can use binary_cross_entropy

**Comment by robinhad** [comment]
Thank you! It worked.
How can I contribute to documentation?
Because there isn't any example of feed forward network with alternative function.

**Comment by cha-zhang** [comment]
CNTK documentation is in this repo: 
https://github.com/MicrosoftDocs/cognitive-toolkit-docs 

It's public so you can create a PR there.
